### ISSUE TITLE
Update the default MATRX case

### ISSUE THREAD

**Issue by thaije** [matrx-software/matrx#285]
**Is your feature request related to a problem? Please describe.**
Always when you use MATRX for the first time with default settings, it starts the (ugly) restaurant case which doesn't give a good impression of what MATRX can do, look like, and how it works.

**Describe the solution you would like**
Pimp the default case to a fancy image and some more interesting agents as a showcase of MATRX. 

**Describe alternatives you have considered**
Use BW4T as the default MATRX case.

**Comment by jwaa** [comment]
As BW4T is now removed from v2.1.0, it is not the best option if we want it updated for v2.1.0. @thaije Any ideas on a good show case? Preferably one that is quickly build so we can still and try to make the release date :P

**Comment by thaije** [comment]
Hmm yeah we could also just make a restaurant teamtask? I'm thinking Overcooked light.
The case: A number of customer agents sit somewhere in the restaurant and order stuff by sending a message to the "waiter" agent. The waiter passes the orders on to the cook agent. The cook agent creates new objects for the orders using the new `AddObject` action, after which these objects are brought to the customer by the waiter. All of the text visible in the chat so the user knows what is going on. Maybe add a human agent that has to remove the dirty dishes from the tables to a `kitchen` (tile) where it disappears. 

I think this case has three important things:
- It shows a teaming task that can be optimized, e.g. by changing the team composition, agent behaviour, etc.
- It showcases some of the possibilities of MATRX
- It sparks a lot of inspiration for ways in which the case can be extended, which are almost all possible in MATRX. E.g. add that customers leave after a certain time not having been helped, customers that come and go after a meal and earn you money, food that can go cold, etc. The user could even completely reimplement `overcooked`.

@jwaa what do you think?

**Comment by jwaa** [comment]
Sounds great! My biggest worry if it is doable in a day. The release is planned tomorrow :p

**Comment by thaije** [comment]
Yeah fair point, the other issues have priority in my opinion so we can see how much time there is left for the case and change accordingly :)
